app/presenters/admin_auction_status_presenter/rejected.rb

```diff
@@ -8,7 +8,7 @@ def body
       'statuses.admin_auction_status_presenter.rejected.body',
       delivery_url: auction.delivery_url,
       rejected_at: rejected_at,
-      winner_email: winner.email
+      winner_name: winner_name
```

Comment:
Nice, but why doesn't this use the `winner_url` method too? Should we fix that while we're doing this?